hw/ip/clkmgr/data/clkmgr_pkg.sv.tpl

```diff
@@ -14,13 +14,23 @@ package clkmgr_pkg;
 % endfor
   } hint_names_e;
 
+  // clocks generated and broadcast
   typedef struct packed {
 % for clk in typed_clocks.all_clocks():
     logic ${clk};
 % endfor
-
   } clkmgr_out_t;
 
+  // clock gating indication for alert handler
+  typedef struct packed {
+<% n_clk = 0 %>\
+% for clk in typed_clocks.all_clocks():
```

Comment:
there might be SOME conflicts between this and #8410, but we'll have to sort it out as it comes.

Reply:
yeah this may need another rebase (depending which one gets merged first).

Reply:
I rebased this onto your changes.